.ci/test_script.sh

```diff
@@ -25,6 +25,7 @@ case "$TEST_TYPE" in
         coverage erase
 
         # Run preliminary tests
+        # coverage run -a "${CI_DIR}/test_quicksetup.py"
```

Comment:
maybe run the test as well ;-)

Reply:
ah no that is on purpose, my comment *meant* to say "I *commented*" the test. I did so because I wanted to see if at least all the rest would work

Reply:
oh sorry, I went directly to the changes and missed your comment. I will look into it